jre/guide/java/java_jre_insecure_prompt/rule.yml

```diff
@@ -0,0 +1,34 @@
+documentation_complete: true
+
+title: 'Prompt User Prior to Executing Mobile Code'
+
+description: |-
+    To ensure that Java prevents mobile code from executing without prompting
+    the user, set <tt>deployment.insecure.jres</tt> to equal
+    <tt>PROMPT</tt> in <tt>/etc/.java/deployment/deployment.properties</tt>.
+
+rationale: |-
+    Mobile code has the potential to cause damage to information systems within
+    an organization if used maliciously. It can execute without explicit action
+    from, or notification to, a user. Mobile code is defined as software modules
+    obtained from remote systems, transferred across a network, and then
+    downloaded and executed on a local system without explicit installation or
+    execution by the recipient. Requiring Java to enforce prompting the user prior
+    to executing mobile code will strenghten the security posture of the system.
```

Comment:
```suggestion
    Mobile code has the potential to cause damage to information systems within
    an organization if used maliciously. It can execute without explicit action
    from, or notification to, a user. Requiring Java to enforce prompting the user prior
    to executing mobile code will strenghten the security posture of the system.
```